src/window.rs

```diff
@@ -799,7 +799,8 @@ impl Window {
     /// ## Platform-specific
     ///
     /// - **macOS:** This locks the cursor in a fixed location, which looks visually awkward.
-    /// - **iOS / Android / Web:** Always returns an [`ExternalError::NotSupported`].
+    /// - **Web:** This uses the pointer lock API
```

Comment:
Are there any quirks of the pointer lock API that warrant the inclusion of this note? To my knowledge, the `## Platform-specific` section is for quirks that are relevant for users of winit, yet I don't see how this is all that useful to know.

Reply:
I can remove that line if that is blocking this PR

Reply:
In light of the failure conditions for [`setPointerCapture`](https://w3c.github.io/pointerevents/#setting-pointer-capture), I believe the following should be documented:
> 3. If the element is not connected [DOM], throw an "InvalidStateError" DOMException.
> 4. If this method is invoked while the element's node document [DOM] has a locked element ([PointerLock] pointerLockElement), throw an "InvalidStateError" DOMException.
> 5. If the pointer is not in the active buttons state or the element's node document is not the active document of the pointer, then terminate these steps.

This text should probably not be copied verbatim.